api/pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>edu.wgu.osmt</groupId>
         <artifactId>osmt-parent</artifactId>
-        <version>2.6.0-SNAPSHOT</version>
```

Comment:
We don't change versions in pom files.. `mvn release` does this for us. These should be reverted.

Reply:
Shouldn't we leave it at least at 3.0.0?